cetmix_tower_server/tests/test_plan.py

```diff
@@ -322,16 +322,50 @@ def test_plan_user_access_rule(self):
         )
         self.assertTrue(self.plan_3)
 
-        # Check what manager can't unlink exisiting plan
+        # Check what manager can't unlink existing plan
         with self.assertRaises(AccessError):
             self.plan_3.with_user(self.user_bob).unlink()
 
         # Add user_bob to group_root
         self.add_to_group(self.user_bob, "cetmix_tower_server.group_root")
 
-        # Check what root can unlink exisiting plan
+        # Check what root can unlink existing plan
         result = self.plan_3.with_user(self.user_bob).unlink()
         self.assertTrue(result)
+        # Remove user_bob from root group to test access to flight plans related
+        # to servers user isn't subscribed
+        self.remove_from_group(
+            self.user_bob,
+            [
+                "cetmix_tower_server.group_root",
+            ],
+        )
+        # Assign plan_1 to self.server_test_1
+        self.plan_2.write({"server_ids": [(6, 0, [self.server_test_1.id])]})
```

Comment:
Please use [wirte_and_invalidate](https://github.com/cetmix/cetmix-tower/blob/e70e241b99cd7bfdc27e389be935b2e2eef34db9/cetmix_tower_server/tests/common.py#L330) method to ensure caches are invalidated.